api/src/auth/drivers/ldap.ts

```diff
@@ -400,9 +364,9 @@ export class LDAPAuthDriver extends AuthDriver {
 
 const handleError = (e: Error) => {
 	if (
-		e instanceof ldap.InappropriateAuthenticationError ||
+		e instanceof ldap.InappropriateAuthError ||
```

Comment:
```suggestion
		e instanceof InappropriateAuthError ||
```
couldnt select all 3 for a suggestion in github